core/math/bsp_tree.cpp

```diff
@@ -577,12 +577,11 @@ BSP_Tree::BSP_Tree(const PoolVector<Face3> &p_faces, real_t p_error_radius) {
 	error_radius = p_error_radius;
 }
 
-BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius) {
-
-	nodes = p_nodes;
-	planes = p_planes;
-	aabb = p_aabb;
-	error_radius = p_error_radius;
+BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius)
```

Comment:
`clang-format` can be a bit uncooperative in my experience. Maybe there's an option to tweak that, but if not in the meantime, I guess we can go with whatever style it enforces (i.e. the one you used). Just fix the other two comments above and it can be merged.